docs/content/preview/yugabyte-cloud/_index.md

```diff
@@ -28,9 +28,9 @@ cascade:
 
 YugabyteDB Managed is a fully managed YugabyteDB-as-a-Service that allows you to run YugabyteDB clusters on Amazon Web Services (AWS), Microsoft Azure, and Google Cloud Platform (GCP).
 
-{{<tip>}}
+{{<lead link="https://status.yugabyte.cloud/">}}
```

Comment:
same thing here.. not sure if we want to have leads or tip in cases where there multiple links.